Commons/bukkit/src/main/java/tc/oc/commons/bukkit/report/ReportCommands.java

```diff
@@ -183,8 +192,8 @@ public void reports(final CommandContext args, final CommandSender sender) throw
                         final Audience audience = audiences.get(sender);
                         audience.sendMessage(new HeaderComponent(title));
                         for(Report report : reportResult.documents()) {
-                            if(report.reported() != null) {
-                                audience.sendMessages(reportFormatter.format(report, crossServer, true));
+                            if(report.reported() != null && onlinePlayers.find(report.reported()).isOnline()) {
```

Comment:
`report.reported() != null && (showOffline || onlinePlayers.find(report.reported()).isOnline())`

Reply:
You need to figure out how to get this to work with the `a` flag